### PR TITLE
[JBIDE-26697] dont watch unnecessary events for Event, PVC and Template

### PR DESCRIPTION
**Pull request by adietish**
# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by adietish** [comment]
@odockal can QE please test this change? We're stopping to watch Builds, Events, PVCs, Templates. In terms of code (and quick manual testing) this should be fine. Builds and Events are not editable (Editors watch for changes since you cannot save a resource that you edit if it was changed on the server). Can you please verify throughout?

File: plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java [hunk 1]
```diff
@@ -39,13 +39,23 @@
 
 public class WatchManager {
 
+	public static final String[] KINDS = new String[] { 
+			//ResourceKind.BUILD,
```
Comment (jeffmaury) [inline plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java]: Why BUILD are not watched ?

Comment (adietish) [reply plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java]: Builds are not editable (no editor that needs to watch them), Properties are not watching any resources, there's no occurrence where I see builds being watched. So why watch them?

**Comment by adietish** [comment]
testPR

File: plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java (hunk 1 above)
Comment (adietish) [reply plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java]: @jeffmaury looking into it in more details I see you being right: added builds in again.

**Pull request merged**